src/code/z_camera_data.c

```diff
@@ -73,8 +80,28 @@ CameraModeValue D_8011A3A0[] = {
     { 0x0003, 9 }, // flags
 };
 
+/**
+ * Set: NORMAL0        Mode: TARGET    Func: PARA1
+ * Set: BOSS_GOMA      Mode: TARGET    Func: PARA1
+ * Set: BOSS_DODO      Mode: TARGET    Func: PARA1
```

Comment:
what about making it look more like a table? ie:
```suggestion
 * Setting        Mode      Function
 * NORMAL0        TARGET    PARA1
 * BOSS_GOMA      TARGET    PARA1
 * BOSS_DODO      TARGET    PARA1
```